src/lib/tasfw-resources/src/PyramidUpdate.cpp

```diff
@@ -440,16 +440,16 @@ void PyramidUpdate::PyramidLoop()
 			dz = 0.0f;
 		}
 
-		if (_state.pyramid.tiltingPyramidMarioOnPlatform == TRUE)
+		if (_state.pyramid.tiltingPyramidMarioOnPlatform == true)
```

Comment:
I left the macros in because it's an integer, but looking at it again there's really no reason not to use boolean literals instead.